### ISSUE TITLE
Add a query for RES policies

### ISSUE THREAD

**Issue by rjplevin** [JGCRI/gcam-core#48]
The query "elec gen by subsector" sums the renewable energy credits used to implement the policy along with the sources of those credits (whatever techs are deemed RE).

Pralit suggesting changing the query to:

```
		<supplyDemandQuery title="elec gen by subsector">
			<axis1 name="subsector">subsector</axis1>
			<axis2 name="Year">physical-output[@vintage]</axis2>
			<xPath buildList="true" dataName="output" group="false" sumAll="false">*[@type='sector' (:collapse:) and (@name='electricity' or @name='elect_td_bld')]/*[@type='subsector' and not (@name='elect_td_bld')]//*[@type='output']/physical-output/node()</xPath>
			<comments/>
		</supplyDemandQuery>
```

to eliminate the final `(:collapse:)`. This shows the `output` as either electricity or your RES market name, avoiding the double-counting. Works for me. 

Since it's not clear that the original query is useful, perhaps these changes should be made in the core.

**Comment by pkyle** [comment]
@pralitp let me know if you want for me to do anything on this.

**Comment by kvcalvin** [comment]
This has been corrected in GCAM v5.2.
